internal/mediainfo/matroska.go

```
@@ -1605,31 +1611,31 @@ func parseMatroskaTrackEntry(buf []byte, segmentDuration float64, durationPrec i
 		}
 		displayWidth := videoInfo.displayWidth
 		displayHeight := videoInfo.displayHeight
-		if displayWidth == 0 && storedWidth > 0 {
+		if displayWidth == 0 && sampledWidth > 0 {
 			crop := videoInfo.cropLeft + videoInfo.cropRight
-			if crop > 0 && crop < storedWidth {
-				displayWidth = storedWidth - crop
+			if crop > 0 && crop < sampledWidth {
+				displayWidth = sampledWidth - crop
```

golangci-lint (2.11.4)

[error] ineffectual assignment to displayWidth

(ineffassign)